frappe/integrations/doctype/paypal_settings/paypal_settings.py

```diff
@@ -146,7 +146,12 @@ def get_payment_url(self, **kwargs):
 			"token": response.get("TOKEN")[0],
 			"correlation_id": response.get("CORRELATIONID")[0]
 		})
-		self.integration_request = create_request_log(kwargs, "Remote", "PayPal", response.get("TOKEN")[0])
```

Comment:
Note for reviewer: this was never supposed to be **Remote**, hence ignored in patch.